allure-rest-assured/build.gradle

```diff
@@ -0,0 +1,20 @@
+description = 'Allure Rest-assured'
+
+apply from: "${gradleScriptDir}/maven-publish.gradle"
+apply from: "${gradleScriptDir}/bintray.gradle"
+apply plugin: 'maven'
+apply plugin: 'java'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
no need to specify the version, we are using dependency management plugin. See https://github.com/allure-framework/allure-java/blob/master/build.gradle#L76

And also we are using single string notation for the deps: `testCompile 'junit:junit'`

Also change the order of dependencies:

1. compile scope, project deps in first place
1. testCompile scope, project deps in first place